### ISSUE TITLE
[feat] Allow experimental features in WebView

### ISSUE THREAD

**Issue by Andful** [tauri-apps/tauri#4897]
### Describe the problem

I would like to enable experimental features on webkit or chromium. Is there a way to do such?

### Describe the solution you'd like

Add documentation on how to enable experimental features if already possible. Otherwise expose a tauri config option to enable experimental browser features.

**Comment by FabianLars** [comment]
If i remember everything correctly we have the following limitations:
- macos: Doesn't support this at all
- windows: Only supports chromium's command line flags. Which for now you have to set via the `WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS` environment flag. Keep in mind that many of these flags only work in non-stable browser versions (webview2 uses the stable channel)
- Linux: Hmm, can't remember the situation, but i _think_ it's similar to macos.

tldr: Really nothing _we_ can do here, so i'm going to close this issue until someone can show that i got that stuff wrong.
